src/Components/Reports/Util.js

```diff
@@ -0,0 +1,36 @@
+import { get } from 'lodash';
+
+export const responseToPDFData = (data) => {
+    const systemsRowsData = [];
+    const rowKeys = ['display_name', 'os', 'performance_utilization.cpu', 'performance_utilization.memory', 'performance_utilization.max_io',
+        'number_of_suggestions', 'state', 'reported_date'];
+
+    data.map((systemItem) => {
+        let rowValueArr = [];
+        rowKeys.map((rowKey) =>{
+            let rowValue =  rowKey === 'reported_date' ? '03 Mar 2022 06:58 UTC'  : get(systemItem, rowKey, '').toString();
+            rowValue = (rowKey === 'performance_utilization.cpu' || rowKey === 'performance_utilization.memory') ? `${rowValue}%` : rowValue;
+            rowValueArr.push(rowValue);
```

Comment:
How `none` value is handled? If nothing or none is sent for a field via an api response , then is it handled here or we need to handle by replacing it with `N/A` or `-` (hyphen)

Reply:
good catch @r14chandra I have fixed the same and added code to handle null and -1 (in case of waiting for data) values. 👍🏻 💯